Index.html

```diff
@@ -0,0 +1,200 @@
+            </p>
+          </div>
+          <div class="home-info-2">
+            <p>
+              This is why the idea of finding love across the dance floor
+              endures – symbolizing that, when we know the true rhythm of our
+              heart, we know the other.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="home" id="home2">
+        <div class="program-title">
+          <h2>Dance Guide</h2>
+        </div>
+        <div class="program-items">
+          <ul>
+            <li>
+              <img
+                src="https://img.icons8.com/color/100/000000/ballerina-full-body.png"
+                alt="logo"
+              />
+              <p>
+                Nobody cares if you can’t dance well. Just get up and dance.
+                Great dancers are great because of their passion.
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/color/96/000000/dancing-party.png"
+                alt="logo"
+              />
+              <p>
+                To dance is to be out of yourself. Larger, more beautiful, more
+                powerful… This is power, it is glory on earth and it is yours
+                for the taking.
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/external-photo3ideastudio-lineal-color-photo3ideastudio/100/000000/external-dancer-celebration-photo3ideastudio-lineal-color-photo3ideastudio.png"
+                alt="logo"
+              />
+              <p>
+                Those who were seen dancing were thought to be insane by those
+                who could not hear the music.
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/officel/80/000000/tango.png"
+                alt="logo"
+              />
+              <p>
+                “Hand in hand, on the edge of the sand, they danced by the light
+                of the moon.”
+              </p>
+            </li>
+            <li>
+              <img
+                src="https://img.icons8.com/dusk/100/000000/dancing.png"
+                alt="logo"
+              />
+              <p>
+                Dance for yourself. If someone understands, good. If not, no
+                matter.
+              </p>
+            </li>
+          </ul>
+          <div class="program-button-container">
+            <button type="submit" class="program-button"><span>See More</span></button>
+          </div>
+        </div>
+        <div class="program-link">
+          <a href="#">See More</a>
+        </div>
+      </section>
+      <section class="feature-section">
+        <h2>Best Dancer's</h2>
+        <ul class="feature">
+        </ul>
+       <div class="showcontainer">
+        <div class="showMore">
+         <span> More</span>
+        </div>
+       </div>
+      </section>
```

Comment:
- [ ] The featured section should:
- - [ ] have the same `background-color` as the design.
- - [ ] It's currently missing the `grid` image behind each performer.
- - [ ] The page has some white space that I am not sure where it's coming from, please investigate this issue and make sure it look similar so the design.
- 
Reference Design
![image](https://user-images.githubusercontent.com/33767951/142339438-e1e363da-0a28-4a48-a88e-3010f2199e84.png)

Your Styling `Home => 786px`
![image](https://user-images.githubusercontent.com/33767951/142339462-6c55b707-6edc-47c3-9b2a-ab5b72a04412.png)